src/js/static-net-filtering.js

```diff
@@ -3691,18 +3692,18 @@ FilterContainer.prototype.freeze = function() {
     // work properly, so defer this until later to allow for reduced delay to
     // readiness when no valid selfie is available.
     if ( this.optimizeTimerId === undefined ) {
-        this.optimizeTimerId = globals.requestIdleCallback(( ) => {
```

Comment:
@gorhill should I rename this to `optimizeTaskId`?

Reply:
>@gorhill should I rename this to optimizeTaskId?

I've taken the liberty and done it now. I checked that it's not accessed from anywhere outside this file.